src/main/resources/application.yml

```diff
@@ -10,6 +10,8 @@ server:
 spring:
   application:
     name: prison-api
+  main:
+    web-application-type: servlet
```

Comment:
needed so that repository tests know that we are in servlet mode and therefore the `HmppsAuthenticationHolder` gets wired up